### DISCUSSION TITLE
how can magick.net add spot color channel

### DISCUSSION THREAD

**Discussion by zhuanzhanfc** [dlemstra/Magick.NET#1041] [Help]
I have tried some methods, but only generate TIF with alpha channel. How can I generate TIF diagram of spot color channel with specified name? Just like the new spot color channel in Photoshop

**Comment by dlemstra** [comment]
There currently is no support for writing extra channels. I also don't know how we should write a spot channel instead of an alpha channel. Is it possible to provide a sample file that has this so we can research this?
